rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/validation/TestCachingValidator.java

```diff
@@ -56,8 +60,12 @@
   private final Configuration conf = new Configuration();
 
   @BeforeMethod
-  public void setUp()
+  public void setUp() throws IOException
   {
+    CacheConfig.setCacheDataDirPrefix(conf, TEST_CACHE_DIR_PREFIX);
+    CacheConfig.setMaxDisks(conf, TEST_MAX_DISKS);
+
+    TestUtil.createCacheParentDirectories(conf, TEST_MAX_DISKS);
   }
 
   @AfterMethod
```

Comment:
Can you also add removeCacheParentDirectories call in the teardown to make sure the directory gets deleted

Reply:
Added

Reply:
Also set the CacheDataDirPrefix to make sure we are deleting the required directory